include/common/unix_sock.h

```diff
@@ -25,4 +25,5 @@
 
 void* accept_unix(void *data);
 int init_sock();
+void dummy_monitor_fn(void* message);
```

Comment:
Now on, please add some comment above the function to tell what the function is supposed to do. 
Since this is new function, please add the comment.

Reply:
Done.